guides/Getting-Started.md

```diff
@@ -258,15 +262,39 @@ Spago can be used to turn our PureScript code into JavaScript suitable for use i
     Build succeeded.
     Bundle succeeded and output file to index.js
 
-All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document. If you try this, you should see the words "Hello, World!" printed to your browser's console.
+All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document.  Try this by creating the following `index.html` file in your `my-project` directory:
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Euler Exercise</title>
+</head>
+
+<body>
+  <div id="container"></div>
+  <script src="./index.js"></script>
+</body>
+
+</html>
+```
+
+See this HTML file work as follows:
+
+1.  Run `npx parcel index.html --open` in a terminal opened to your `my-project`.
+1.  In the newly-opened browser, browse `localhost:1234`.  You will probably see a blank window.
+1.  Open a **debugger console**.
+1.  Observe the console contains the Euler exercise result string `The answer is 233168`.
```

Comment:
@milesfrain At this point, the transition to the explanation of the `index.js` file is abrupt (my doing).  What occurs to me is I'd like to see adding instructions to write the Euler result to a text control that shows in the HTML page.  Then provide the smooth transition to the `index.js` file only explain its sending the result to the **HTML text control** rather than to the **console**.  Would this be worthwhile?

Reply:
I'd support adding a "Rendering in Browser" section after this one, that shows how to render the answer as HTML text to the page and describing why `spago run` breaks.

Reply:
I'd rather not, actually; I think this should be beyond the scope of this guide, especially because there are quite a few options for this, and none of them seem ideal. I don't think the low-level DOM API bindings are a good fit because they're very verbose and people are unlikely to be using them in a real project either; alternatively you could use one of the web UI libraries like halogen or react-basic, but then you have to privilege one library over the others, and I think such documentation is better placed in those individual libraries anyway.

Reply:
Ah, I'd forgot you had opened an issue for this already (#325), let's have that discussion there then.

Reply:
I think it's fine to say that if you use the browser to run your PureScript code that output will be written to the browser console instead of being written to your terminal. I agree that this minimal getting started guide shouldn't introduce any UI libraries or use the DOM bindings.